.github/CODEOWNERS

```diff
@@ -14,6 +14,8 @@
 /cmd/agent/app/integrations*.go         @DataDog/prodsec @DataDog/agent-integrations @DataDog/agent-core
 /cmd/cluster-agent/                     @DataDog/container-integrations
 
+/docs/                                  @DataDog/baklava
```

Comment:
could you add `@Datadog/agent-core` as well?

Otherwise Agent Core will not own this section anymore. If that was intentional let's discuss this :)